### ISSUE TITLE
Conda installation, files missing from site-packages/nbformat/v4 folder 

### ISSUE THREAD

**Issue by JeMorriso** [jupyter/notebook#5944]
## Description

Installed in fresh Conda environment, Jupyter Notebook cannot open a new notebook, because files are missing from `site-packages/nbformat/`. I can copy the `v4` folder specified in the FileNotFoundError below from a jupyter notebook installed via `pipenv`, which does contain the missing file. If I do so, then the Conda version works without raising the below exception.

Output from the running Jupyter instance on trying to open a new file:

```bash
[E 15:42:25.221 NotebookApp] Uncaught exception POST /api/contents (127.0.0.1)
    HTTPServerRequest(protocol='http', host='127.0.0.1:8888', method='POST', uri='/api/contents', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/tornado/web.py", line 1704, in _execute
        result = await result
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/tornado/gen.py", line 769, in run
        yielded = self.gen.throw(*exc_info)  # type: ignore
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/notebook/services/contents/handlers.py", line 202, in post
        yield self._new_untitled(path, type=type, ext=ext)
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/tornado/gen.py", line 762, in run
        value = future.result()
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/notebook/services/contents/handlers.py", line 154, in _new_untitled
        model = yield maybe_future(self.contents_manager.new_untitled(path=path, type=type, ext=ext))
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/notebook/services/contents/manager.py", line 397, in new_untitled
        return self.new(model, path)
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/notebook/services/contents/manager.py", line 416, in new
        model['content'] = new_notebook()
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/nbformat/v4/nbbase.py", line 164, in new_notebook
        validate(nb)
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/nbformat/v4/nbbase.py", line 39, in validate
        return validate(node, ref=ref, version=nbformat)
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/nbformat/validator.py", line 268, in validate
        for error in iter_validate(nbdict, ref=ref, version=version,
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/nbformat/validator.py", line 303, in iter_validate
        validator = get_validator(version, version_minor, relax_add_props=relax_add_props)
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/nbformat/validator.py", line 57, in get_validator
        schema_json = _get_schema_json(v, version=version, version_minor=version_minor)
      File "/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/nbformat/validator.py", line 94, in _get_schema_json
        with open(schema_path) as f:
    FileNotFoundError: [Errno 2] No such file or directory: '/home/jerry/miniconda3/envs/jlab-test/lib/python3.9/site-packages/nbformat/v4/nbformat.v4.5.schema.json'
[W 15:42:25.225 NotebookApp] Unhandled error
[E 15:42:25.225 NotebookApp] {
      "Host": "127.0.0.1:8888",
      "Connection": "keep-alive",
      "Content-Length": "19",
      "Accept": "application/json, text/javascript, */*; q=0.01",
      "X-Requested-With": "XMLHttpRequest",
      "X-Xsrftoken": "2|2621ada7|70b1890755f351e683552fa6bf854565|1610667737",
      "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.101 Safari/537.36",
      "Content-Type": "application/json",
      "Sec-Gpc": "1",
      "Origin": "http://127.0.0.1:8888",
      "Sec-Fetch-Site": "same-origin",
      "Sec-Fetch-Mode": "cors",
      "Sec-Fetch-Dest": "empty",
      "Referer": "http://127.0.0.1:8888/tree",
      "Accept-Encoding": "gzip, deflate, br",
      "Accept-Language": "en-GB,en-US;q=0.9,en;q=0.8",
      "Cookie": "username-127-0-0-1-8888=\"2|1:0|10:1610667737|23:username-127-0-0-1-8888|44:YTBjN2I1ZTZhNWY2NDI2NDkyYzcyMjQ2ZjQ3N2RmOGE=|38b8532f271550215bfc5bf32635566eba735369a75ca25cc0af16c0baf62778\"; _xsrf=2|2621ada7|70b1890755f351e683552fa6bf854565|1610667737"
    }
[E 15:42:25.226 NotebookApp] 500 POST /api/contents (127.0.0.1) 5.880000ms referer=http://127.0.0.1:8888/tree
```

## Reproduce

1. Create new Conda environment: `conda create -n jnb-test --override-channels --strict-channel-priority -c notebook`
2. Launch notebook: `jupyter notebook`
3. Try and open a new notebook.
4. See the output in the terminal window.

## Expected behavior

New notebook to open.

## Context

- Operating System and version: EndeavourOS Linux 1.4
- Browser and version: Brave V1.18.78
- Jupyter Notebook version: 6.2.0

<details><summary>Troubleshoot Output</summary>
<pre>
$PATH:
	/home/jerry/miniconda3/envs/jnb-test/bin
	/home/jerry/miniconda3/bin
	/home/jerry/miniconda3/condabin
	PYENV_ROOT/bin
	/home/jerry/.npm-global/bin
	/home/jerry/.local/bin
	/usr/local/bin
	/usr/bin
	/bin
	/usr/local/sbin
	/usr/lib/jvm/default/bin
	/usr/bin/site_perl
	/usr/bin/vendor_perl
	/usr/bin/core_perl
	/home/jerry/.fzf/bin

sys.path:
/home/jerry/miniconda3/envs/jnb-test/bin
/home/jerry/miniconda3/envs/jnb-test/lib/python39.zip
/home/jerry/miniconda3/envs/jnb-test/lib/python3.9
/home/jerry/miniconda3/envs/jnb-test/lib/python3.9/lib-dynload
/home/jerry/.local/lib/python3.9/site-packages
/home/jerry/miniconda3/envs/jnb-test/lib/python3.9/site-packages

sys.executable:
/home/jerry/miniconda3/envs/jnb-test/bin/python

sys.version:
3.9.1 | packaged by conda-forge | (default, Jan 10 2021, 02:55:42)
[GCC 9.3.0]

platform.platform():
Linux-5.10.6-arch1-1-x86_64-with-glibc2.32

which -a jupyter:
/home/jerry/miniconda3/envs/jnb-test/bin/jupyter
/home/jerry/miniconda3/bin/jupyter

pip list:
Package Version
----------------------------- ----------
alabaster 0.7.12
appdirs 1.4.4
argon2-cffi 20.1.0
async-generator 1.10
attrs 20.3.0
Babel 2.9.0
backcall 0.2.0
backports.functools-lru-cache 1.6.1
beautifulsoup4 4.9.3
bleach 3.2.1
buku 4.5
certifi 2020.12.5
cffi 1.14.4
chardet 4.0.0
colorama 0.4.4
cryptography 3.3.1
decorator 4.4.2
defusedxml 0.6.0
distlib 0.3.1
docutils 0.16
entrypoints 0.3
filelock 3.0.12
html5lib 1.1
idna 2.10
imagesize 1.2.0
importlib-metadata 3.4.0
ipykernel 5.4.2
ipython 7.19.0
ipython-genutils 0.2.0
jedi 0.18.0
Jinja2 2.11.2
jsonschema 3.2.0
jupyter-client 6.1.11
jupyter-core 4.7.0
jupyterlab-pygments 0.1.2
MarkupSafe 1.1.1
mistune 0.8.4
nbclient 0.5.1
nbconvert 6.0.7
nbformat 5.1.0
nest-asyncio 1.4.3
notebook 6.2.0
packaging 20.8
pandocfilters 1.4.2
parso 0.8.1
pexpect 4.8.0
pickleshare 0.7.5
pip 20.3.3
pipenv 2020.11.15
prometheus-client 0.9.0
prompt-toolkit 3.0.10
psutil 5.8.0
ptyprocess 0.7.0
pycparser 2.20
Pygments 2.7.3
pyparsing 2.4.7
pyrsistent 0.17.3
pyte 0.8.0
python-dateutil 2.8.1
pytz 2020.5
pyzmq 21.0.0
requests 2.25.1
Send2Trash 1.5.0
setuptools 51.1.2
six 1.15.0
snowballstemmer 2.0.0
soupsieve 2.1
sphinxcontrib-applehelp 1.0.2
sphinxcontrib-devhelp 1.0.2
sphinxcontrib-htmlhelp 1.0.3
sphinxcontrib-jsmath 1.0.1
sphinxcontrib-qthelp 1.0.3
sphinxcontrib-serializinghtml 1.1.4
terminado 0.9.2
testpath 0.4.4
thefuck 3.30
tornado 6.1
traitlets 5.0.5
urllib3 1.26.2
virtualenv 20.3.0
virtualenv-clone 0.5.4
wcwidth 0.2.5
webencodings 0.5.1
wheel 0.36.2
zipp 3.4.0

conda list: # packages in environment at /home/jerry/miniconda3/envs/jnb-test: # # Name Version Build Channel
\_libgcc_mutex 0.1 conda_forge conda-forge
\_openmp_mutex 4.5 1_gnu conda-forge
argon2-cffi 20.1.0 py39h3811e60_2 conda-forge
async_generator 1.10 py_0 conda-forge
attrs 20.3.0 pyhd3deb0d_0 conda-forge
backcall 0.2.0 pyh9f0ad1d_0 conda-forge
backports 1.0 py_2 conda-forge
backports.functools_lru_cache 1.6.1 py_0 conda-forge
bleach 3.2.1 pyh9f0ad1d_0 conda-forge
ca-certificates 2020.12.5 ha878542_0 conda-forge
certifi 2020.12.5 py39hf3d152e_1 conda-forge
cffi 1.14.4 py39he32792d_1 conda-forge
decorator 4.4.2 py_0 conda-forge
defusedxml 0.6.0 py_0 conda-forge
entrypoints 0.3 pyhd8ed1ab_1003 conda-forge
importlib-metadata 3.4.0 py39hf3d152e_0 conda-forge
importlib_metadata 3.4.0 hd8ed1ab_0 conda-forge
ipykernel 5.4.2 py39hef51801_0 conda-forge
ipython 7.19.0 py39hef51801_0 conda-forge
ipython_genutils 0.2.0 py_1 conda-forge
jedi 0.18.0 py39hf3d152e_2 conda-forge
jinja2 2.11.2 pyh9f0ad1d_0 conda-forge
jsonschema 3.2.0 py_2 conda-forge
jupyter_client 6.1.11 pyhd8ed1ab_1 conda-forge
jupyter_core 4.7.0 py39hf3d152e_0 conda-forge
jupyterlab_pygments 0.1.2 pyh9f0ad1d_0 conda-forge
ld_impl_linux-64 2.35.1 hea4e1c9_1 conda-forge
libffi 3.3 h58526e2_2 conda-forge
libgcc-ng 9.3.0 h5dbcf3e_17 conda-forge
libgomp 9.3.0 h5dbcf3e_17 conda-forge
libsodium 1.0.18 h36c2ea0_1 conda-forge
libstdcxx-ng 9.3.0 h2ae2ef3_17 conda-forge
markupsafe 1.1.1 py39h3811e60_3 conda-forge
mistune 0.8.4 py39h3811e60_1003 conda-forge
nbclient 0.5.1 py_0 conda-forge
nbconvert 6.0.7 py39hf3d152e_3 conda-forge
nbformat 5.1.0 pyhd8ed1ab_0 conda-forge
ncurses 6.2 h58526e2_4 conda-forge
nest-asyncio 1.4.3 pyhd8ed1ab_0 conda-forge
notebook 6.2.0 py39hf3d152e_0 conda-forge
openssl 1.1.1i h7f98852_0 conda-forge
packaging 20.8 pyhd3deb0d_0 conda-forge
pandoc 2.11.3.2 h7f98852_0 conda-forge
pandocfilters 1.4.2 py_1 conda-forge
parso 0.8.1 pyhd8ed1ab_0 conda-forge
pexpect 4.8.0 pyh9f0ad1d_2 conda-forge
pickleshare 0.7.5 py_1003 conda-forge
pip 20.3.3 pyhd8ed1ab_0 conda-forge
prometheus_client 0.9.0 pyhd3deb0d_0 conda-forge
prompt-toolkit 3.0.10 pyha770c72_0 conda-forge
ptyprocess 0.7.0 pyhd3deb0d_0 conda-forge
pycparser 2.20 pyh9f0ad1d_2 conda-forge
pygments 2.7.3 pyhd8ed1ab_0 conda-forge
pyparsing 2.4.7 pyh9f0ad1d_0 conda-forge
pyrsistent 0.17.3 py39h3811e60_2 conda-forge
python 3.9.1 hffdb5ce_3_cpython conda-forge
python-dateutil 2.8.1 py_0 conda-forge
python_abi 3.9 1_cp39 conda-forge
pyzmq 21.0.0 py39hea8fd45_0 conda-forge
readline 8.0 he28a2e2_2 conda-forge
send2trash 1.5.0 py_0 conda-forge
setuptools 49.6.0 py39hf3d152e_3 conda-forge
six 1.15.0 pyh9f0ad1d_0 conda-forge
sqlite 3.34.0 h74cdb3f_0 conda-forge
terminado 0.9.2 py39hf3d152e_0 conda-forge
testpath 0.4.4 py_0 conda-forge
tk 8.6.10 h21135ba_1 conda-forge
tornado 6.1 py39h3811e60_1 conda-forge
traitlets 5.0.5 py_0 conda-forge
tzdata 2020f he74cb21_0 conda-forge
wcwidth 0.2.5 pyh9f0ad1d_2 conda-forge
webencodings 0.5.1 py_1 conda-forge
wheel 0.36.2 pyhd3deb0d_0 conda-forge
xz 5.2.5 h516909a_1 conda-forge
zeromq 4.3.3 h58526e2_3 conda-forge
zipp 3.4.0 py_0 conda-forge
zlib 1.2.11 h516909a_1010 conda-forge

conda env:
name: jnb-test
channels: - conda-forge - defaults
dependencies: - \_libgcc_mutex=0.1=conda_forge - \_openmp_mutex=4.5=1_gnu - argon2-cffi=20.1.0=py39h3811e60_2 - async_generator=1.10=py_0 - attrs=20.3.0=pyhd3deb0d_0 - backcall=0.2.0=pyh9f0ad1d_0 - backports=1.0=py_2 - backports.functools_lru_cache=1.6.1=py_0 - bleach=3.2.1=pyh9f0ad1d_0 - ca-certificates=2020.12.5=ha878542_0 - certifi=2020.12.5=py39hf3d152e_1 - cffi=1.14.4=py39he32792d_1 - decorator=4.4.2=py_0 - defusedxml=0.6.0=py_0 - entrypoints=0.3=pyhd8ed1ab_1003 - importlib-metadata=3.4.0=py39hf3d152e_0 - importlib_metadata=3.4.0=hd8ed1ab_0 - ipykernel=5.4.2=py39hef51801_0 - ipython=7.19.0=py39hef51801_0 - ipython_genutils=0.2.0=py_1 - jedi=0.18.0=py39hf3d152e_2 - jinja2=2.11.2=pyh9f0ad1d_0 - jsonschema=3.2.0=py_2 - jupyter_client=6.1.11=pyhd8ed1ab_1 - jupyter_core=4.7.0=py39hf3d152e_0 - jupyterlab_pygments=0.1.2=pyh9f0ad1d_0 - ld_impl_linux-64=2.35.1=hea4e1c9_1 - libffi=3.3=h58526e2_2 - libgcc-ng=9.3.0=h5dbcf3e_17 - libgomp=9.3.0=h5dbcf3e_17 - libsodium=1.0.18=h36c2ea0_1 - libstdcxx-ng=9.3.0=h2ae2ef3_17 - markupsafe=1.1.1=py39h3811e60_3 - mistune=0.8.4=py39h3811e60_1003 - nbclient=0.5.1=py_0 - nbconvert=6.0.7=py39hf3d152e_3 - nbformat=5.1.0=pyhd8ed1ab_0 - ncurses=6.2=h58526e2_4 - nest-asyncio=1.4.3=pyhd8ed1ab_0 - notebook=6.2.0=py39hf3d152e_0 - openssl=1.1.1i=h7f98852_0 - packaging=20.8=pyhd3deb0d_0 - pandoc=2.11.3.2=h7f98852_0 - pandocfilters=1.4.2=py_1 - parso=0.8.1=pyhd8ed1ab_0 - pexpect=4.8.0=pyh9f0ad1d_2 - pickleshare=0.7.5=py_1003 - pip=20.3.3=pyhd8ed1ab_0 - prometheus_client=0.9.0=pyhd3deb0d_0 - prompt-toolkit=3.0.10=pyha770c72_0 - ptyprocess=0.7.0=pyhd3deb0d_0 - pycparser=2.20=pyh9f0ad1d_2 - pygments=2.7.3=pyhd8ed1ab_0 - pyparsing=2.4.7=pyh9f0ad1d_0 - pyrsistent=0.17.3=py39h3811e60_2 - python=3.9.1=hffdb5ce_3_cpython - python-dateutil=2.8.1=py_0 - python_abi=3.9=1_cp39 - pyzmq=21.0.0=py39hea8fd45_0 - readline=8.0=he28a2e2_2 - send2trash=1.5.0=py_0 - setuptools=49.6.0=py39hf3d152e_3 - six=1.15.0=pyh9f0ad1d_0 - sqlite=3.34.0=h74cdb3f_0 - terminado=0.9.2=py39hf3d152e_0 - testpath=0.4.4=py_0 - tk=8.6.10=h21135ba_1 - tornado=6.1=py39h3811e60_1 - traitlets=5.0.5=py_0 - tzdata=2020f=he74cb21_0 - wcwidth=0.2.5=pyh9f0ad1d_2 - webencodings=0.5.1=py_1 - wheel=0.36.2=pyhd3deb0d_0 - xz=5.2.5=h516909a_1 - zeromq=4.3.3=h58526e2_3 - zipp=3.4.0=py_0 - zlib=1.2.11=h516909a_1010
prefix: /home/jerry/miniconda3/envs/jnb-test
t/pre>

</details>

**Comment by Zsailer** [comment]
This is an issue in nbformat. It was fixed in `nbformat==5.1.2`. Try upgrading:
```
pip install -U nbformat==5.1.2
```

**Comment by JeMorriso** [comment]
Well that was easy. I ran `conda update --all` which updated nbformat to 5.1.1 and it's working now. Thanks for your help, I figured conda would have by default installed the latest versions of everything possible, but evidently not.
